### PR TITLE
Use Bazel 0.18 for development

### PR DESCRIPTION
**Pull request by mboes**
Includes switching to a newer Nixpkgs and a small compiler version
update.

### CODE REVIEW

File: nixpkgs/default.nix [hunk 1]
```diff
@@ -1,4 +1,4 @@
 import (fetchTarball {
-   url = "https://github.com/NixOS/nixpkgs/archive/9f768ff43cece413e922f93cf861a28eaa969c41.tar.gz";
-   sha256 = "0dnfqh1hm0b3wwghqxhj358jhlprdrx9da1kd20g9635g7j0qs10";
+   url = "https://github.com/NixOS/nixpkgs/archive/ef4d78adedcfbb69392e7634bc6912c019b88bcc.tar.gz";
```
Comment (guibou) [inline nixpkgs/default.nix]: I suggest to add a comment with the date of the nixpkgs checkout.

Comment (mboes) [reply nixpkgs/default.nix]: Good idea.

**Comment by Profpatsch** [comment]
I think the darwin CI instance is stuck.

**Comment by guibou** [comment]
@Profpatsch yes, it was building `ghc` `8.2.2` from source ;)

**Pull request merged**